Created timestamps-2022-09-08 for [Ufuncs and dtypes: New possibiliti… (#27)
numpy/numpy-newsletter: youtube-timestamps/timestamps-2022-06-04.md

0:00 Statement on the NumPy Code of Conduct

01:04 Introduction of the speaker

02:10 Self-introduction of each participant

> 06:05 Beginning of slide presentation and general benefit of NumPy

Reviewer: ```suggestion
06:51 Beginning of slide presentation and general benefit of NumPy
```
I changed this to align with the exact time the slide appears on the screen.

08:03 Self-introduction of the speaker

09:01 Accessing current presentation and other resources

10:02 Why use NumPy in Mechanical Engineering courses?

12:54 Exercises: questions to participants and speaker's comments

27:55 Diagram on Learner's Journey

> 29:45 Presentation of an open educational resource in Computational Mechanics

Reviewer: ```suggestion
29:45 Reviewing an open educational resource in Computational Mechanics
```

35:26 Revisiting the Learner's Journey

> 26:14 NumPy in Mechanical Engineering

Reviewer: ```suggestion
38:35 NumPy in Mechanical Engineering
```
Seems like even though the slide was on screen there was a separate discussion going on before the talk about the topic on screen.
